mdt.py

```diff
@@ -96,29 +97,26 @@ def run(path, recursive, src_lang, dest_lang, output, debug):
         if (path.endswith("." + src_lang + ".md") or (
                     src_lang == const.LANG_EN and "." not in path.split("/")[-1].rstrip(".md"))):
             translate_page(
-                path.removesuffix('.md') if src_lang == const.LANG_EN else path.removesuffix('.' + src_lang + '.md'),
+                re.sub('\.md$', '', path) if src_lang == const.LANG_EN else re.sub('.' + src_lang + '.md', '', path),
```

Comment:
ここ、エスケープを忘れてるように見える！
```suggestion
                re.sub('\.md$', '', path) if src_lang == const.LANG_EN else re.sub('\.' + src_lang + '.md', '', path),
```

Reply:
天才！！全然気づかなかった。